test.py

```
@@ -0,0 +1,27 @@
+import os,sys 
+
+def DoSomething(x,y):
+  if x==None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)